keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -729,6 +736,87 @@ that might indicate a serious problem?
 
 #### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
+We have unit tests testing the "rollback" scenario as we do test with positive and negative cases of the feature.
```

Comment:
Can you link them?

Are those really upgrade/downgrade tests (i.e. change the FG in the middle of the test) or actually test the cases where feature is on/off?

Reply:
test cases with feature on/off.  I have some tests I am working on but I did not get them in alpha.  

I want to add some integration tests for upgrade and rollback.  I will also add an integration test for upgrade/downgrade/upgrade.  

Is it sufficient to add this to beta promotion?  Or is this blocking the promotion?

Reply:
I have upgrade and downgrade tests in the integration tests but I am running into some troubles in the integration tests on testing the upgrade -> downgrade -> upgrade path.